tests/abe/cover_crypt.test.ts

```diff
@@ -1,24 +1,125 @@
-import { CoverCryptHybridDecryption } from "../../src/crypto/abe/core/hybrid_crypto/cover_crypt/decryption"
-import { CoverCryptHybridEncryption } from "../../src/crypto/abe/core/hybrid_crypto/cover_crypt/encryption"
-import { CoverCryptKeyGeneration } from "../../src/crypto/abe/core/keygen/cover_crypt"
-import { CoverCryptDemoKeys } from "./cover_crypt_demo_keys"
-import { EncryptionDecryptionDemo } from "./hybrid_crypto_utils"
+import { CoverCrypt, hexDecode, Policy, PolicyAxis } from "index"
+
+const TOP_SECRET_MKG_FIN_USER_ACCESS_POLICY =
+  "Security Level::Top Secret && (Department::MKG || Department::FIN)"
+
+const TOP_SECRET_MKG_FIN_USER = hexDecode(
+  "2a403d8f9e0c4441c0d074a0cd925f95ff47cdbd5e7704160960ec7ac2c27a0a1f33ac06dfcf0d65008e9a3b0af987be4a86d79781196b24dae47da44f5c620a0a378fee44df891d837e1bdd78be3aa2de174830a6148e72d2c35c15efde2353004f3d59e3ea6ec4dc314741a6c0e18895d82046bf1813e3781c4c2afafb0b2d00d0ae65693bcfb22c10e06deabe00656b36fb7778913ace143edffa0f0fd1de08eaff457c46e180661c7cc08b7bf3665940e46b402ed4a08ba48ee73fd08ee605e5f5347a2420609a5d45876e09ca1543e8005fbd8441d115cd5529425a571a0e62405280f32e7e5bae0e25d25668aeaecd8be317ed3a1bfb9618ca812b5ec50ac26e5e241a4b2646ef9954a9c2b06d6da556057c9fc1dbc0274ade63627d67071eafd82d43228e007b861961888e62b107d0c16f54b4750b16c80f04369a170e488e38b3719bffaa0597ffa2d1ba57558e2a1b3c487f872a31bc6498b8bb0403f2c1f93b6c7281c6822878e88e30b2e1ba9d656e9c229697a798591158d3de0e"
+)
+
+// User decryption key with access policy: "Security Level::Medium Secret && Department::MKG"
+const MEDIUM_SECRET_MKG_USER_ACCESS_POLICY =
+  "Security Level::Medium Secret && Department::MKG"
+
+// Plaintext example is: My secret message
+const PLAINTEXT = hexDecode("4d7920736563726574206d657373616765")
+
+// Hybrid encrypted data: ABE attributes are ['Security Level::Low Secret', 'Department::HR', 'Department::FIN']
```

Comment:
I don't even know how this list is supposed to be translated into attributes. The intent should be something like that
```suggestion
// Hybrid encrypted data: ABE access policy is "Security Level::Low Secret && (Department::HR || Department::FIN)"
```

... but to my understanding this list is supposed to be translated as
 ``` rust
// Hybrid encrypted data: ABE access policy is
// "Security Level::Low Secret && Department::HR && Department::FIN"
```
which is supposed to be an empty set